rules/the-happiness-equation/rule.md

```diff
@@ -31,11 +31,11 @@ Here are 10 tips for how you can manage unfairness.
 1. **Gotta speak up** - See [Do you know to speak up?](/speak-up)
 2. **Happiness is relative** - Unhappiness can come from comparisons with others.  Instead compare yourself with your day yesterday.
 3. **Get one thing in life, lose another** - Can't have everything.  When you are saying 'Yes' to one thing, you are saying 'No' to another.
-4. **The 'Happiness Equation'**   
-  Happiness = Expectations - Reality.<br>
-  If you want to be happier, then:   
-     - Reduce your expectations
-     - Increase your reality
+4. **The 'Happiness Equation'**
+  Happiness = Reality - Expectations.<br>
```

Comment:
I disagree with this equation because it makes no sense.
Happiness = Expectations (10) - Reality (6)
Happiness = 4

Happiness = Reality (6) - Expectation (10)
Happiness = -4

When expectations are low, it makes even less sense.
Happiness = Expectations (2) - Reality (8)
Happiness = -6

Happiness = Reality (8) - Expectations (2)
Happiness = 6